src/main/java/ch/njol/skript/expressions/ExprFacing.java

```diff
@@ -37,49 +37,58 @@
 import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Facing")
-@Description("The facing of an entity or block, i.e. exactly north, south, east, west, up or down (unlike <a href='#ExprDirection'>direction</a> which is the exact direction, e.g. '0.5 south and 0.7 east')")
-@Examples({"# makes a bridge",
-		"loop blocks from the block below the player in the horizontal facing of the player:",
-		"\tset loop-block to cobblestone"})
-@Since("1.4")
+@Description({
+	"The facing or opposite facing of an entity or block, i.e. exactly north, south, east, west, up or down",
+	"unlike <a href='/expressions.html#ExprDirection'>direction</a> which is the exact direction, e.g. '0.5 south and 0.7 east'",
+	"NOTE: Changing the opposite facing has the same result of changing the actual facing."
+})
+@Examples({
+	"# Makes a bridge",
+	"loop blocks from the block below the player in the horizontal facing of the player:",
+	"",
+	"# Get the block's face you're looking at",
+	"on right click:",
+		"\tsend \"You're looking at %opposite facing of player% side of %type of targeted block%.\""
+})
+@Since("1.4, INSERT VERSION (opposite)")
 public class ExprFacing extends SimplePropertyExpression<Object, Direction> {
 
 	static {
-		register(ExprFacing.class, Direction.class, "(1¦horizontal|) facing", "livingentities/blocks");
+		register(ExprFacing.class, Direction.class, "[:opposite] [:horizontal] facing", "livingentities/blocks");
 	}
 	
-	private boolean horizontal;
-	
+	private boolean horizontal, opposite;
+
 	@Override
-	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parseResult) {
-		horizontal = parseResult.mark == 1;
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		horizontal = parseResult.hasTag("horizontal");
+		opposite = parseResult.hasTag("opposite");
 		return super.init(exprs, matchedPattern, isDelayed, parseResult);
 	}
 	
-	@SuppressWarnings("deprecation")
 	@Override
 	@Nullable
-	public Direction convert(final Object o) {
+	@SuppressWarnings("deprecation")
+	public Direction convert(Object o) {
 		if (o instanceof Block) {
 			BlockData data = ((Block) o).getBlockData();
 			if (data instanceof org.bukkit.block.data.Directional) {
-				return new Direction(((org.bukkit.block.data.Directional) data).getFacing(), 1);
+				org.bukkit.block.data.Directional directional = (org.bukkit.block.data.Directional) data;
+				return new Direction(opposite ? directional.getFacing().getOppositeFace() : directional.getFacing(), 1);
 			}
 			return null;
 		} else if (o instanceof LivingEntity) {
-			return new Direction(Direction.getFacing(((LivingEntity) o).getLocation(), horizontal), 1);
+			BlockFace facing = opposite ? Direction.getFacing(((LivingEntity) o).getLocation(), horizontal).getOppositeFace() : Direction.getFacing(((LivingEntity) o).getLocation(), horizontal);
```

Comment:
I think a variable would be good to store
```java
Direction.getFacing(((LivingEntity) o).getLocation(), horizontal)
```
Since it seems to be used by both "options" of the ternary